part-time-schedule/week-1.md

```diff
@@ -0,0 +1,87 @@
+
+{% hint style="warning" %}
+Don't run `bitcoind` without specifying `regtest` (or `signet`) as the network in the configuration file (`bitcoin.conf`) or as a command line argument (see `bitcoind --help`). Otherwise you will start synchronising blocks for mainnet which is over 400 GB! At this point, you don't even need to run bitcoind just yet.
+{% endhint %}
+
+## Goals
+
+* Discuss and understand bitcoin philosophy
+* Become comfortable and proficient in setting up bitcoin developer environments
+* Be able to run and interface with Bitcoin Core and LND
+  * Understand signet and regtest
+  * Connect Bitcoin Core and LND programmatically
+  * Interface with Bitcoin Core and LND using the CLI tools
+  * Programmatic control of bitcoind and LND
+  * Comfortable with running the full test suite(s) of Bitcoin Core
+* Build a toy bitcoin wallet using a library
+  * Add additional functionality of your choosing
+* Develop a robust understanding of bitcoin topics both in theory and in practice:
+  * Random number generation
+  * Keys and key material
+  * Derivation paths
+  * Address types
+  * Coin selection
+* Gain experience with developing and using a Bitcoin wallet UI
+
+## Monday & Tuesday
+
+1. Introduction to Qala
+2. Programme structure
+3. Icebreakers
+4. Bitcoin Philosophy
+5. Why we bitcoin
+
+## Wednesday - Friday
+
+### Resources
+
+| Name                                                               | Link                                                                                                                                                                                                   |
+| ------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| BIP0032                                                            | [https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)                                                                       |
+| TABConf 2021 Building Your own bitcoin wallet with BitcoinJS       | [https://youtu.be/Bwz2P2hPVpk](https://youtu.be/Bwz2P2hPVpk)                                                                                                                                           |
+| Bitstein - Setting up a bitcoin lightning network test environment | [https://medium.com/@bitstein/setting-up-a-bitcoin-lightning-network-test-environment-ab967167594a](https://medium.com/@bitstein/setting-up-a-bitcoin-lightning-network-test-environment-ab967167594a) |
+
+### Exercises
+
+1. Read the Mastering Lightning book appendix on bitcoin transactions.
+   * The username:password to access the book is: `qala:lightning`
```

Comment:
I'd replace that with this link: https://github.com/lnbook/lnbook/blob/develop/appendix_bitcoin_fundamentals_review.asciidoc#bitcoin-transactions